cluster/gce/gci/configure-helper-common.sh

```diff
@@ -1808,16 +1808,21 @@ function start-kube-apiserver {
     local max_mutating_request_inflight="-1"
 
     # If the cluster is large, increase max-requests-inflight limit in apiserver.
-    if [[ "${NUM_NODES}" -gt 10000 ]]; then
-       max_request_inflight=5000
-       max_mutating_request_inflight=2000
+    if [[ "${NUM_NODES}" -ge 25000 ]]; then
+       max_request_inflight=40000
+       max_mutating_request_inflight=20000
     else
-      if [[ "${NUM_NODES}" -gt 3000 ]]; then
-        max_request_inflight=3000
-        max_mutating_request_inflight=1000
-      elif [[ "${NUM_NODES}" -gt 500 ]]; then
-        max_request_inflight=1500
-        max_mutating_request_inflight=500
+      if [[ "${NUM_NODES}" -gt 10000 ]]; then
+        max_request_inflight=5000
```

Comment:
Where is 5000/2000 come from? I don't have data in hand. Not sure whether 20K cluster can have mutating request restricted to 2000. For 25K cluster, it spikes to 6K. I would not think 2K is enought for 20K cluster.

Reply:
Check the history and found 5000/2000 was added by Vinay when move scale-out from POC to master. For 40000/20000 is the data that we are used for current 1*1*25K and 2*2*25K